web/src/OrganizationEditPage.js

```diff
@@ -436,6 +436,16 @@ class OrganizationEditPage extends React.Component {
             }} />
           </Col>
         </Row>
+        <Row style={{marginTop: "20px"}} >
+          <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 19 : 2}>
+            {Setting.getLabel(i18next.t("organization:Use email as username"), i18next.t("organization:Use email as username - Tooltip"))} :
```

Comment:
Use email as username -> Use Email as username for everywhere

Reply:
Done